(#1500) Makes default value for REMEMBER_PARAMETERS_BETWEEN_SAMPLES and queue_manager.centringMethod configurable

mockoocy opened the pull request:
This PR makes settings mentioned in the title configurable. Also makes them behave in an expected manner, that is:
- they persist between refreshes (`centringMethod` was reset on the frontend after refresh)
- are reset properly when user logs out

The default values are set with `default_centring_method` and `default_remember_parameters_between_samples`. 


It also fixes the problem with `remember parameters between samples` checkbox. When It was changed in the settings, although boolean value was sent by client, it registered a bool-casted-to-string value, that is `"False"` or `"True"`.

Closes #1500

marcus-oscarsson commented on mxcubeweb/core/components/queue.py:
```diff
@@ -2078,6 +2080,20 @@ def init_queue_settings(self):
 
+        self.app.REMEMBER_PARAMETERS_BETWEEN_SAMPLES = (
+            HWR.beamline.queue_manager.get_property(
+                "default_remember_parameters_between_samples", False
```
Looks good in general, do we need to call it `default_remember_parameters_between_samples` with the `default` prefix, would not `remember_parameters_between_samples` be sufficient ? I guess one understands that the user can change the option ?

mockoocy replied:
Both can work. I think that providing the `default` prefix is a good idea for fields that may be modified by the user, to avoid some awkward bugs with e.g., some fields not resetting - but that's only me preference :)

fabcor-maxiv replied:
There are two levels of "default" here, aren't there?

1. `False` is the default fall back value if `default_remember_parameters_between_samples` is not set, right?
2. and then `default_remember_parameters_between_samples` is also a "default" because it can be overridden via the checkbox in the UI, correct? It is the default value when starting MXCuBE.

marcus-oscarsson replied:
:), yes. That can be a bit confusing. I would not make things too complicated. I don't think we need to the have the fallback/default argument configurable (`False` in this case) . I also don't think we need to keep the `default` prefix, some parameters are changeable by the users other are not, do we need to distinguish between them ? In that case a comment is perhaps more suitable.

mockoocy replied:
Okay, maybe too many defaults there :D will remove the prefix then.

Edit: removed the prefix and included small comment on why it's done that way.

marcus-oscarsson commented on mxcubeweb/core/components/queue.py:
```diff
@@ -2077,6 +2079,21 @@ def init_queue_settings(self):
+            )
+        )
+
+        default_centring_method_as_string = HWR.beamline.queue_manager.get_property(
```
I guess the same thing here then ?

mockoocy replied:
and now... in this case it would not work as the `centring_method` property is being set directly as 
`HWR.beamline.queue_manager.centring_method = ...`, so if we'd want to make it behave as the other settings we need to explicitly store it.

marcus-oscarsson replied:
I meant for the naming with the `default` prefix :)

mockoocy replied:
Oh, changed then :)

marcus-oscarsson replied:
:)

mockoocy committed:
Makes queue settings persist between refreshes.
also makes remember_parameters_between_samples and centring_method have their default values configurable
